rotkehlchen/globaldb/schema.py

```diff
@@ -148,6 +160,59 @@
 );
 """
 
+DB_V3_CREATE_COMMON_ASSETS_DETAILS = """
+CREATE TABLE IF NOT EXISTS common_asset_details(
+    identifier TEXT PRIMARY KEY NOT NULL COLLATE NOCASE,
+    name TEXT,
+    symbol TEXT,
+    coingecko TEXT,
+    cryptocompare TEXT,
+    forked TEXT,
+    FOREIGN KEY(forked) REFERENCES assets(identifier) ON UPDATE CASCADE
+);
+"""
+
+DB_V3_CREATE_ASSETS = """
+CREATE TABLE IF NOT EXISTS assets (
+    identifier TEXT PRIMARY KEY NOT NULL COLLATE NOCASE,
+    type CHAR(1) NOT NULL DEFAULT('A') REFERENCES asset_types(type),
+    started INTEGER,
+    swapped_for TEXT,
+    common_details_id TEXT,
+    FOREIGN KEY(common_details_id) REFERENCES common_asset_details(identifier) ON UPDATE CASCADE
+);
+"""
+
+DB_V3_CREATE_EVM_TOKENS = """
+CREATE TABLE IF NOT EXISTS evm_tokens (
+    identifier TEXT PRIMARY KEY NOT NULL COLLATE NOCASE,
+    token_type TEXT NOT NULL,
+    chain TEXT NOT NULL,
+    address VARCHAR[42] NOT NULL,
+    decimals INTEGER,
+    protocol TEXT
+);
+"""
+
+DB_V3_CREATE_ASSETS_EVM_TOKENS = """
+CREATE TABLE IF NOT EXISTS assets_evm_tokens(
```

Comment:
Didn't you say this is probably not needed? Don't remember how that conversation ended.

Reply:
Yes, indeed I didn't make the insert